examples/firehose/firehose-enable-logs/variables.tf

```diff
@@ -21,6 +21,12 @@ variable "logs_enable" {
   default     = false
 }
 
+variable "metric_enable" {
+  description = "Enble sending metrics to Coralogix"
```

Comment:
Enble -> Enable, also on the logs_enable above

Reply:
Done